lnrpc/signrpc/signer_server.go

```diff
@@ -31,8 +36,33 @@ const (
 	// SubServerConfigDispatcher instance recognize this as the name of the
 	// config file that we need.
 	subServerName = "SignRPC"
+
+	// MuSig2SessionIDSize is the size of a MuSig2 session ID. It is the
+	// length of the combined public key and the two public nonces, all of
+	// them being compressed points.
+	MuSig2SessionIDSize = 99
```

Comment:
Ah I see this sort of combines the notions of the session+context into one. If we wanted to get away with a smaller value there, then we can just hash these items and use a single 32 byte value here.

Reply:
Good idea with the hashing, added that.